### ISSUE TITLE
Fix pino bundling in docs

### ISSUE THREAD

**Issue by jmrossy** [hyperlane-xyz/v3-docs#80]
https://discord.com/channels/935678348330434570/1224477833879879840/1225497208158158971

https://getpino.io/#/docs/bundling

https://github.com/pinojs/pino/blob/06df1df8a33262a0a0a268087f1a30cd44be6850/lib/tools.js#L290

![Image](https://github.com/hyperlane-xyz/v3-docs/assets/710751/8031d227-2a15-452b-b234-f543f63129d7)

**Comment by jmrossy** [comment]
Fixed in https://github.com/hyperlane-xyz/v3-docs/pull/79/commits/5aa79acc9b8162c0eee1ce4868521708df779003

**Comment by jmrossy** [comment]
For posterity: 
I've tried getting the pino webpack plugin to work in Docusaurus but it's a dead end. The pino plugin breaks the CSS minification step... no idea how those are related. It also breaks the dev server because Docusaurus tries to run it in the browser (what!?). 

Fortunately, the reason for the fuss was to get the SDK working in the docs to import the chain metadata and addresses. There's a shiny new package meant just for that! I've replaced the SDK with the Registry in @yorhodes's PR here: https://github.com/hyperlane-xyz/v3-docs/pull/79/commits/5aa79acc9b8162c0eee1ce4868521708df779003

If we ever do want to use the SDK or Utils in the docs, it'll likely require replacing Pino first.
